conda_forge_tick/auto_tick.xsh

```diff
@@ -121,6 +121,25 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
     return migrate_return, pr_json
 
 
+def add_rebuild(migrators):
+    """Adds rebuild migrators.
+
+    Parameters
+    ----------
+    migrators : list of Migrator
+        The list of migrators to run.
+
+    """
+
+    pygraph = copy.deepcopy(gx)
+    for node, attrs in gx.node.items():
+        if 'python' not in attrs['req']:
```

Comment:
python should be in host.

Reply:
Not all the native python packages (which are not noarch) have been migrated to the new syntax so they don't have host's. Do we only care about python at build time?